sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -538,7 +538,9 @@ static inline kernel_pid_t _next_hop_l2addr(uint8_t *l2addr, uint8_t *l2addr_len
     }
 #endif
 #if defined(MODULE_GNRC_NDP_NODE)
-    iface = gnrc_ndp_node_next_hop_l2addr(l2addr, l2addr_len, iface, dst, pkt);
+    if (iface <= KERNEL_PID_UNDEF) {
```

Comment:
Args… I only just now realized, that you also deactivate next hop determination for the case that the interface was given this way :/